From aebf66e59d5de5d546f94ffa6e954ff8fb760320 Mon Sep 17 00:00:00 2001
From: Sachin Prabhu <sprabhu@redhat.com>
Date: Mon, 28 Jul 2014 12:16:30 -0400
Subject: [fs] cifs: Add missing end of line termination to some cifs messages

Message-id: <1406549822-10376-38-git-send-email-sprabhu@redhat.com>
Patchwork-id: 86749
O-Subject: [RHEL7.1 PATCH 37/69] [bz1082049]Add missing end of line termination to some cifs messages
Bugzilla: 1082049
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Steve French <smfrench@gmail.com>

Signed-off-by: Steve French <smfrench@gmail.com>
Signed-off-by: Gregor Beck <gbeck@sernet.de>
Reviewed-by: Jeff Layton <jlayton@redhat.com>
(cherry picked from commit ebcc943c11f48617a7536a132c64d2637075e407)
Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/cifs/cifssmb.c b/fs/cifs/cifssmb.c
index dea7f0e..5cc5004 100644
--- a/fs/cifs/cifssmb.c
+++ b/fs/cifs/cifssmb.c
@@ -4005,7 +4005,7 @@ QFileInfoRetry:
  rc = SendReceive(xid, tcon->ses, (struct smb_hdr *) pSMB,
     (struct smb_hdr *) pSMBr, &bytes_returned, 0);
  if (rc) {
-  cifs_dbg(FYI, "Send error in QPathInfo = %d\n", rc);
+  cifs_dbg(FYI, "Send error in QFileInfo = %d", rc);
  } else {  /* decode response */
   rc = validate_t2((struct smb_t2_rsp *)pSMBr);
 
@@ -4174,7 +4174,7 @@ UnixQFileInfoRetry:
  rc = SendReceive(xid, tcon->ses, (struct smb_hdr *) pSMB,
     (struct smb_hdr *) pSMBr, &bytes_returned, 0);
  if (rc) {
-  cifs_dbg(FYI, "Send error in QPathInfo = %d\n", rc);
+  cifs_dbg(FYI, "Send error in UnixQFileInfo = %d", rc);
  } else {  /* decode response */
   rc = validate_t2((struct smb_t2_rsp *)pSMBr);
 
@@ -4258,7 +4258,7 @@ UnixQPathInfoRetry:
  rc = SendReceive(xid, tcon->ses, (struct smb_hdr *) pSMB,
     (struct smb_hdr *) pSMBr, &bytes_returned, 0);
  if (rc) {
-  cifs_dbg(FYI, "Send error in QPathInfo = %d\n", rc);
+  cifs_dbg(FYI, "Send error in UnixQPathInfo = %d", rc);
  } else {  /* decode response */
   rc = validate_t2((struct smb_t2_rsp *)pSMBr);
 
-- 
1.7.1